packages/core/template/project/index.md

```diff
@@ -0,0 +1,28 @@
+---
+  layout: default.md
+  title: ""
+---
+
+# ProjectEx
+
+[![CI Status](https://github.com/MarkBind/markbind/workflows/CI/badge.svg)](https://github.com/MarkBind/markbind/actions)
+[![codecov](https://codecov.io/gh/MarkBind/markbind/branch/master/graph/badge.svg)](https://codecov.io/gh/MarkBind/markbind)
+
+<img src="images/SeEduLogo.png" width="200px">
+
+<box type="tip" seamless>
+Can insert your app's onboarding screen or representative picture here.
+</box>
+
```

Comment:
What if you add in a "How to use this template" box? Something that stands out really obviously.